libraries/jevmodal/media/css/jevmodal.css

```diff
@@ -161,7 +162,7 @@
 	max-height:100%;
 }
 .jevmodal .modal-dialog {
-    height: 100%;
+    height: 95%;
```

Comment:
Actually Kenneth did in any of his custom css files and told me, but you will be able to find it easily with CSS inspector. I did not apply any changes on his JEvents files as he already took care of it.

Access details will be in your inbox in 5 minutes.